Automattic-Tracks-iOS/Services/EventLoggingUploadQueue.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+open class EventLoggingUploadQueue {
+
+    private let fileManager = FileManager.default
```

Comment:
I think it'd be better to take this as a constructor parameter instead so the caller can customize it to their needs. It'd also make this class more testable.

Reply:
Good call – adjusted with https://github.com/Automattic/Automattic-Tracks-iOS/commit/810b5c9124ad66e6eff3d08c8e714ebf460bf496.